content/blog/2016/2016-11-20-what-jvm-versions-are-running-jenkins-the-return.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "What JVM versions are running Jenkins? 2016 Update!"
+tags:
+- general
+- statistics
+author: batmat
+---
+
+CAUTION: **Disclaimer:** What follows contains some opinions or statements which may not be seen as purely factual or neutral. Note that this represents by no mean the general position of the Jenkins governance board. This is solely my opinion as a contributor based on the data I gathered, and what I feel from the feedback of the community at large.
```

Comment:
s/**Disclaimer**//

There's already the `CAUTION` admonition here.

I also suggest putting this after the first paragraph, and then moving the `TIP` section to the bottom of the post